### PR TITLE
Publicize: Improve the text/usability of Twitter thread options

### PR DESCRIPTION
**Pull request by pento**
This PR includes a couple of tweaks to the Twitter options panel, making it more usable.

- The style and copy has been tweaked to more closely match the post visibility radio options in Core.
- The panel is now hidden when there are no Twitter accounts selected.

Screen Shot 2020-10-07 at 2 25 02 pm Screen Shot 2020-10-07 at 2 25 10 pm

#### Changes proposed in this Pull Request:
* Tweaks to the Publicize Twitter options panel.

#### Jetpack product discussion
p58i-9vv-p2#comment-47632

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Open the Publicize panel in a post.
* Ensure that the options look correct.
* Ensure that the options are only visible when one or more Twitter accounts is enabled.

#### Proposed changelog entry for your changes:
* Publicize: Improve the usability of the Twitter thread options.

### CODE REVIEW

File: extensions/blocks/publicize/twitter/options.js
Comment (pento) [inline extensions/blocks/publicize/twitter/options.js:83]: I'm not super excited about this workaround, but `RadioControl` doesn't expose a prop to do what we want, and `PostVisibility` in Gutenberg works around this by re-implementing the entire thing.

**Comment by rickybanister** [comment]
The settings look much better, thanks @pento 

cc @lcollette for a second look

**Review by jeherve — DISMISSED** [review]

**Comment by pento** [comment]
@lcollette: Just checking, did you have anything to add before this is merged?

**Review by jeherve — APPROVED** [review]
Re-approving after a rebase.

**Comment by pento** [comment]
r215027-wpcom

**Pull request merged**
